caleb-lab/.gitignore

```diff
@@ -0,0 +1,58 @@
+# Logs
```

Comment:
**TODO FOR LOST POINTS**

Top-level comment to let you know you need a ``README.md`` file, aka docs, per the instructions:

```
Write docs on how to get the project running
Write docs on how to connect to the server
```